src/context/CollectionsAPI/api.tsx

```diff
@@ -98,3 +98,25 @@ export const uploadPhoto = async (
     return resp && get(resp, "collectionAddUpload");
   }
 };
+
+export const renameCollection = async (
+  context: AppContextState,
+  collectionId: number,
+  collectionName: string,
+  isPublic: boolean
+) => {
+  const collectionEditMutation = `
+    mutation collectionEdit($id: Int!, $name: String, $isPublic: Boolean) {
+      collectionEdit(id: $id, name: $name, isPublic: $isPublic )
+      }`;
+  const resp = await collectionsGraphqlRequest(
+    context,
+    collectionEditMutation,
+    {
+      id: collectionId,
+      name: collectionName,
+      isPublic: isPublic,
```

Comment:
I don't think we need to pass isPublic here. Since the variable is not required (it isn't listed as `Boolean!` in the docs), we can omit it. That way we won't have to keep track of isPublic and just pass the new name.